cmd/tableprinters/image.go

```diff
@@ -35,7 +33,7 @@ func (t *TablePrinter) ImageTable(data []*models.V1ImageResponse, wide bool) ([]
 		if wide {
 			usedBy = strings.Join(image.Usedby, "\n")
 		}
-		if !showUsage {
+		if !viper.GetBool("show-usage") {
```

Comment:
Why inside the loop ?

Reply:
Ah, that's why it was there. Brought back.